render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/solvers/affine/AffineAlignBlockWorker.java

```diff
@@ -123,8 +123,9 @@ public AffineAlignBlockWorker(
 
 		// NOTE: if you choose to stitch first, you need to pre-align, otherwise, it's OK to use the initial alignment for each tile
 		if (stitchFirst && parameters.preAlign() == PreAlign.NONE) {
-			LOG.error("Since you choose to stitch first, you must pre-align with Translation or Rigid.");
-			throw new RuntimeException("Since you choose to stitch first, you must pre-align with Translation or Rigid.");
+			final String msg = "Since you choose to stitch first, you must pre-align with Translation or Rigid.";
+			LOG.error(msg);
+			throw new RuntimeException(msg);
```

Comment:
I suggest changing this to an `IllegalArgumentException`, adding `throws IllegalArgumentException` to the constructor declaration, and a throws javadoc comment.